src/blog/2023/03/comparinig-dashboards.md

```diff
@@ -0,0 +1,179 @@
+
+### uibuilder - 1st place
+
+A search on [DuckDuckGo for the term 'node red ui builder'](https://duckduckgo.com/?q=node+red+ui+builder&t=h_&ia=web) returns the correct custom node. When searching for the custom node in the palette manager there is only one result, this is great as users are very likely to install what they were searching for. Once you've found the correct custom node, the installation takes just a few moments using the palette manager.
+
+### Dashboard - 2nd place
+
+As Dashboard is currently the most popular solution to build dashboards in Node-RED, it's very easy to find both in search engines and in the Node-RED interface. A [DuckDuckGo search for 'node red dashboard'](https://duckduckgo.com/?q=node+red+dashboard&t=h_&ia=web) brings up the correct custom node. Finding this custom node in Node-RED's palette manager is not quite as easy, at the time of writing it's the third from top result for the search term 'dashboard'. Some users might not select the intended item from the palette manager on first attempt. However, once you have found the correct custom node, installation is easy and takes just a few moments.
+
+### FlexDash - 3rd place
+
+When [searching on DuckDuckGo for 'node red flexdash'](https://duckduckgo.com/?q=node+red+flexdash&t=h_&ia=web) the top result is the Node-RED website for the custom node. The issue with this, and this is also a problem when searching in the palette manager, is the project 'FlexDash' is apparently not what we actually need to install. When reading the readme for the project on Github it says *'You most likely do not want to explicitly install this package, you want to install the [core widgets](https://github.com/flexdash/node-red-fd-corewidgets), which will bring in this package and more and will provide a usable whole'.*
+
+Credit to the developers for adding in this helpful text but I suspect most users will start off by installing FlexDash then later discover that was not the correct way to proceed. It would be great if the custom node which needs to be installed was the one called 'FlexDash' in my opinion.
+
+This problem is compounded by their being no help file at all for 'FlexDash' showing up on the Node-RED web site. I suspect that may be a deliberate attempt to get users to reconsider if they are looking at the right custom node but it was still a confusing start for me and I suspect many users will have a similar experience.
+
+When setting up FlexDash, one thing that wasn't immediately obvious was that I needed to restart Node-RED before the custom node showed in the palette. This step is [covered in the docs](https://flexdash.github.io/docs/quick-start/#installing-flexdash-in-node-red) but I suspect a lot of users will get stuck working out why the palette manager says the custom node is installed but nothing new has been added to the palette.
+
+I believe a few improvements to the install process could make FlexDash a much more popular custom node.
```

Comment:
May wish to add something about a resolution being actively discussed here: https://github.com/node-red/node-red/issues/569#issuecomment-1460556901

Reply:
Added to both places I've mentioned the install problem, thanks!